app/org/sagebionetworks/bridge/services/AuthenticationService.java

```diff
@@ -185,6 +202,8 @@ public void signOut(final UserSession session) {
         if (session != null) {
             AccountId accountId = AccountId.forId(session.getStudyIdentifier().getIdentifier(), session.getId());
             accountDao.deleteReauthToken(accountId);
+            String reauthCacheKey = getReauthCacheKey(session.getStudyIdentifier().getIdentifier(), session.getReauthToken());
+            cacheProvider.removeObject(reauthCacheKey);
```

Comment:
Should we add a check to the sign out unit test?

Reply:
Added